pandas/tests/groupby/transform/test_transform.py

```diff
@@ -1195,3 +1195,27 @@ def test_transform_lambda_indexing():
         ),
     )
     tm.assert_frame_equal(result, expected)
+
+
+def test_categorical_and_not_categorical_key():
+    # GH 32494
```

Comment:
can you provide a 1-liner here about this this test is actually testing

Reply:
Have added a description (longer than one line though, is that OK?)